src/ng2-auto-complete.component.ts

```diff
@@ -5,23 +5,24 @@ import {
   Output,
   OnInit,
   ViewEncapsulation,
-  EventEmitter
+  EventEmitter,
+  ViewChild
 } from "@angular/core";
-import { Ng2AutoComplete } from "./ng2-auto-complete";
+import {Ng2AutoComplete} from "./ng2-auto-complete";
 
```

Comment:
Coding style conflict. Could you please provide clear reason why use {XXX} instead of { XXX }?